ui/shared/components/form/XHRUploaderField.jsx

```diff
@@ -50,23 +50,23 @@ const hasUploadedFile = value => (value && value.uploadedFileId ? undefined : 'F
 export const validateUploadedFile = value => uploadedFileHasErrors(value) || hasUploadedFile(value)
 export const warnUploadedFile = value => value && value.warnings && (value.warnings.length ? value.warnings : undefined)
 
-const UploaderFormField = React.memo(({ name, required, onChange, normalize, ...props }) => (
+const UploaderFormField = React.memo(({ name, required, onChange, parse, ...props }) => (
   <Field
     name={name}
     validate={required ? validateUploadedFile : uploadedFileHasErrors}
     warn={warnUploadedFile}
     uploaderProps={props}
     component={UploaderFieldComponent}
     onChange={onChange}
-    normalize={normalize}
+    parse={parse}
   />
 ))
 
 UploaderFormField.propTypes = {
   name: PropTypes.string.isRequired,
   required: PropTypes.bool,
   onChange: PropTypes.func,
-  normalize: PropTypes.func,
+  parse: PropTypes.func,
```

Comment:
I thought about that, but because the function signature changed it was important to me that I went through and checked every single case where this is defined to make sure it meets the new function signature, and changing the name was an easy way to make sure I hd caught everything. I think given that its already done I'd rather not change it back, as then we have to support 2 possible names for this parameter in perpetuity, which could get confusing to maintain